Clicker/Quarks/PollQuark.swift

```diff
@@ -64,12 +63,7 @@ struct GetPoll: ClickerQuark {
     func process(element: Element) throws -> Poll {
         switch element {
         case .node(let node):
-            guard let id = node["id"].int, let text = node["text"].string, let results = node["results"].dictionaryObject, let type = node["type"].string, let shared = node["shared"].bool else {
-                throw NeutronError.badResponseData
-            }
-            let questionType: QuestionType = (type == Identifiers.multipleChoiceIdentifier) ? .multipleChoice : .freeResponse
-            let state: PollState = shared ? .shared : .ended
-            return Poll(id: id, text: text, results: results, type: questionType, state: state)
+            return PollParser.parseItem(json: node)
```

Comment:
Way to outsource this logic to these parsers and make CreatePoll a lot more readable. Nice!